app/mailers/room_ticket_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class RoomTicketMailer < ApplicationMailer
+  def send_tdx_ticket
+    @message = params[:message]
+    @submitter = params[:submitter]
+    @room = params[:room]
+    @date = params[:date]
+    @building = @room.floor.building
+
+    subject = "Issue for Room " + @room.room_number + " in " + @building.name + ", "  + @date
```

Comment:
instead of @date you can use show_date(DateTime.now).
show_date is a helper method to display a date.
Add `include ApplicationHelper`to the file.